snooty.toml

```
@@ -14,3 +14,4 @@ mms = ["v4.2", "v4.0", "v3.6", "v3.4", "v2.0", "v1.8", "v1.6", "v1.5", "v1.4", "
 mongocli = ["v0.5.0", "v0.2.0", "v0.1.0", "v0.0.4", "v0.0.3"]
 ruby-driver = ["v2.3", "v2.2", "v2.0", "v1.x"]
 spark-connector = ["v2.0", "v1.1"]
+cloud-docs-osb = ["master"]
```

I believe this should be equal to the project name!

```suggestion
atlas-open-service-broker = ["master"]
```

---

I was wondering about this! I guessed instead of looking harder 😅  Fixed!